src/components/Terminal/Terminal.tsx

```diff
@@ -100,12 +101,41 @@ class Terminal extends React.Component<TerminalProps, {}> {
 
   componentDidMount() {
     this.props.rootStore.start();
+    this.handlerMouseUpTransparentDiv();
+  }
+
+  handlerMouseUpTransparentDiv() {
+    const mosaicSplit = document.getElementsByClassName(
+      'mosaic-split -column'
```

Comment:
it will affect **all** vertical splitters and break the chart

you may check Orderbook/Trade log splitter